.github/workflows/cloud-admin.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Cloud Admin to Netlify
+on:
+  push:
+    tags:
+      - "*"
+    branches: ["main"]
+    paths:
+      - ".github/workflows/cloud-admin.yml"
+      - "web-admin/**"
+      - "web-local/**"
+env:
+  NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+  NETLIFY_SITE_ID: ${{ fromJSON('["e73ac785-882e-425a-8fc8-5432528bb374","afb26917-7a60-47d9-81dc-c0babbb0d9a9"]')[ startsWith(github.ref, 'refs/tags/v') == 'true' ] }}
+  DEPLOY_SITE: ${{ fromJSON('["ui.rilldata.io","ui.rilldata.com"]')[ startsWith(github.ref, 'refs/tags/v') == 'true' ] }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
I'm not sure if this is a problem or not, but I think the Applications team has all been on Node 16. (We should probably upgrade as mentioned [here](https://github.com/rilldata/rill-developer/issues/1916))

Reply:
we have node 16 in the tests too
https://github.com/rilldata/rill-developer/blob/main/.github/workflows/web-test.yml#L31-L34

will move all to 16 then